### ISSUE TITLE
[A11y]Ensures aria-hidden elements do not contain focusable elements

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9299]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1656379
Environment Details:
Application Name: NuGet Service
#URL: https://www.nuget.org/packages/MySqlConnector
Windows version: Windows 10
ChromiumEdge Version: 105.0.1343.42 (Official Build) (64-bit)

Repro Steps:
Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
Launch the application URL: https://www.nuget.org/packages/MySqlConnector
Run the Accessibility insights for web 

Issue: 
Ensures aria-hidden elements do not contain focusable elements (aria-hidden-focus - https://accessibilityinsights.io/info-examples/web/aria-hidden-focus)

Target application: NuGet Gallery | MySqlConnector 2.1.13 - https://www.nuget.org/packages/MySqlConnector

Element path: .prefix-reserve-title

Snippet: <span class="prefix-reserve-title" aria-hidden="true">

How to fix:
Fix all of the following:
  Focusable content should have tabindex='-1' or be removed from the DOM

Environment: Microsoft Edge version 106.0.1370.52

Note:
This appears to show up on many pages. As such, fixing it in the template should suffice to address all instances of this.
